iOS/FVM/Framework/DamagedParts/Providers/DamagedPartsNamesProvider.swift

```diff
@@ -15,11 +15,35 @@
 import Foundation
 
 internal class DamagedPartsNamesProvider : DamagedPartsNamesProviderProtocol {
-    private var validPartsNames: [String]
-    
-    init(validPartsNames: [String]) {
-        self.validPartsNames = validPartsNames
-    }
+    private let validPartsNames: [String] = [
```

Comment:
Now, when you change model you have to change this array. I don't get it.

Reply:
Nope, because every model should be compliant with supported part names defined in schema.json file

Reply:
so define it in schema.json and load here

Reply:
Will do

Reply:
done